docs/reference/release-notes/highlights.asciidoc

```diff
@@ -1,6 +1,5 @@
-[chapter]
```

Comment:
@brianseeders we recently moved some of these files around in the docs nav so we need to keep the `[chapter]` id going forward and the title should start at H1